scripts/config.py

```diff
@@ -42,6 +42,10 @@ def __init__(self):
         self.fast_token_limit = int(os.getenv("FAST_TOKEN_LIMIT", 4000))
         self.smart_token_limit = int(os.getenv("SMART_TOKEN_LIMIT", 8000))
 
+        self.chroma_db_directory = os.getenv("CHROMA-DB-DIRECTORY")
```

Comment:
Please use underscores instead of dashes for environment variables

Reply:
done and done